app/pb_kits/playbook/pb_table/_table.jsx

```diff
@@ -1,61 +1,39 @@
-import React from 'react';
-import classnames from 'classnames';
-import PropTypes from "prop-types";
+/* @flow */
 
-const propTypes = {
-  children: PropTypes.oneOfType([
-      PropTypes.arrayOf(PropTypes.node),
-      PropTypes.node
-  ]),
-  className: PropTypes.string,
-  container: PropTypes.bool,
-  dark: PropTypes.bool,
-  disable_hover: PropTypes.bool,
-  single_line: PropTypes.bool,
-  size: PropTypes.oneOf(["sm", "md", "lg"])
-};
+import React, { type Node } from 'react'
+import classnames from 'classnames'
 
-const defaultProps = {
-  container: true,
-  dark: false,
-  disable_hover: false,
-  single_line: false,
-  size: "sm"
-};
-
-class Table extends React.Component {
-  render() {
-    const {
-      children,
-      className,
-      container,
-      dark,
-      disable_hover,
-      single_line,
-      size
-    } = this.props;
+type TableProps = {
+  children: Array<Node> | Node,
+  className: string,
+  container: boolean,
+  dark: boolean,
+  disableHover: boolean,
+  singleLine: boolean,
```

Comment:
@rafbgarcia - I am on the fence as to what we should do here. Technically, as you have mentioned, this is a breaking change. Since there isn't any use in Nitro I am wondering about two difference options:

# Option #1 (technically the right way)

- Keep the old props beside the new props
- check for either version of the props
- this keeps from introducing a breaking change
- add temporary ESlint override
- open a ticket in GH targeting 4.0.0 release which will remove the old props

# Option #2 (technically the wrong way)

- Keep the changes in this PR and release to `3.2.0` since doing so will not break Nitro

WDYT?

Reply:
If we are going to release `3.2.0` we should go with #1

Reply:
Ok I agree

Reply:
Ahh I just read this. Let me know where we land so I can version properly!